Client/Frontend/Browser/URLBarView.swift

```diff
@@ -74,6 +74,7 @@ class URLBarView: UIView {
                 // Cancel any pending/in-progress animations related to the progress bar
                 self.progressBar.setProgress(1, animated: false)
                 self.progressBar.alpha = 0.0
+                self.progressBar.isHidden = true
```

Comment:
We already have alpha for determining if the progress bar is hidden, don't need to add this I don't think.